var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -55,9 +56,14 @@ class Adios(AutotoolsPackage):
     variant('zlib', default=True, description='Enable zlib transform support')
     variant('bzip2', default=False, description='Enable bzip2 transform support')
     variant('szip', default=False, description='Enable szip transform support')
-    variant('zfp', default=False, description='Enable ZFP transform support')
+    variant('zfp', default=True, description='Enable ZFP transform support')
+    variant('sz', default=True, description='Enable SZ transform support')
```

Comment:
yes, but please do **not** make this *enabled by default* yet, but set the default to `False`. With that upcoming changes in `sz`'s repo until the next `sz` stable release (tagged version) do not break the default build of ADIOS.